test/Clang/clang_default_environment.py

```diff
@@ -25,13 +25,18 @@
 __revision__ = "__FILE__ __REVISION__ __DATE__ __DEVELOPER__"
 
 import TestSCons
+from SCons.Environment import Base
 
 _exe = TestSCons._exe
 test = TestSCons.TestSCons()
 
 if not test.where_is('clang'):
     test.skip_test("Could not find 'clang', skipping test.\n")
 
+platform = Base()['PLATFORM']
```

Comment:
Doesn't the above where_is('clang') take care of this?

Reply:
default environment on windows will have the vcvarsall.bat run which will not taken into account the original environment and path that test.where_is sees